compiler/rustc_middle/src/ty/util.rs

```diff
@@ -1250,7 +1250,7 @@ pub struct AlwaysRequiresDrop;
 
 /// Normalizes all opaque types in the given value, replacing them
```

Comment:
This doc comment should probably be adjusted, too. It still mentions normalization.